site/recent-changes.md

```diff
@@ -1,8 +1,8 @@
-# Recent Changes to the Oracle WebLogic Server Kubernetes Operator
+# Recent changes to the Oracle WebLogic Server Kubernetes Operator
 
-This page tracks recent changes to the operator, especially ones that introduce backward incompatibilities.
+This document tracks recent changes to the operator, especially ones that introduce backward incompatibilities.
 
-| Date | Introduces Backward Incompatibilities | Change |
+| Date | Introduces backward incompatibilities | Change |
 | --- | --- | --- |
 | March 20, 2018 | yes | See [Name Changes](name-changes.md).  Several files and input parameters have been renamed.  This affects how operators and domains are created.  It also changes generated Kubernetes artifacts, therefore customers must recreate their operators and domains.
 | April 4, 2018 | yes | See [Name Changes](name-changes.md).  Many Kubernetes artifact names and labels have changed. Also, the names of generated YAML files for creating a domain's PV and PVC have changed.  Because of these changes, customers must recreate their operators and domains.
```

Comment:
| --- | --- | --- |
 | May 7, 2018 | no | Add support for Dynamic Clusters, add support for Apache HTTP Server, add support for Voyager Ingress Controller, add support for PV in NFS storage for multi node environments.

Reply:
Done.

Reply:
Rosemary, as part of your change, could you remove the references to name-changes.md from recent-changes.md, and git rm name-changes.md too?  Thanks, -Tom

Reply:
Will do.

Reply:
Tom, do you want me to remove the cross references to name-changes.md only or to also remove the accompanying text, for example, "Several files and input parameters have been renamed.  This affects how operators and domains are created.  It also changes generated Kubernetes artifacts, therefore customers must recreate their operators and domains." Similar text occurs in the March 20 and April 4 changes.